anomaly_detector/adaptors/base_model_adaptor.py

```diff
@@ -0,0 +1,17 @@
+"""Base Model Adaptor Class - As we try different models and developers can contribute their custom code."""
+
+from abc import ABCMeta, abstractmethod
```

Comment:
is abc a python thing?

Reply:
yea it is. its part of python-3  https://www.python.org/dev/peps/pep-3119/